src/alchemlyb/preprocessing/subsampling.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 from pymbar.timeseries import statisticalInefficiency
 from pymbar.timeseries import detectEquilibration
+from pymbar.timeseries import subsampleCorrelatedData
```

Comment:
import together (combine all three lines, no idea why they were separate)
```python
from pymbar.timeseries import statisticalInefficiency, detectEquilibration, subsampleCorrelatedData
```

Reply:
@orbeckst yes, changed to one line